tools/test/audio/process_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+set -e
+
+# This process_test.sh is shell script wrapper for process_test.m
+# This needs octave octave-signal octave-io packages.
+
+main ()
+{
+	echo "process_test, " Component="$1", "$2", "$3", "$4", "$5"
+	octave -q --eval "[n_fail]=process_test('$1', $2, $3, $4, $5);exit(n_fail)"
```

Comment:
Should it fail if no passed case?

https://github.com/thesofproject/sof/blob/aed2770e4f6dd48d229a77dffbbb9cb924ea8ad4/tools/test/audio/process_test.m#L136-L140

Reply:
I don't think n_fail=0 && n_pass=0 is possible case, isn't it?

Reply:
It happened when I forgot to run `./scripts/rebuild-testbench.sh` XD